docs/extensions/IconMarkupExtensions.md

```diff
@@ -0,0 +1,209 @@
+
+### Syntax
+
+**XAML**
+
+```xml
+<CommandBar>
+
+    <!--Before-->
+    <AppBarButton>
+        <AppBarButton.Icon>
+            <SymbolIcon Symbol="Play"/>
+        </AppBarButton.Icon>
+    </AppBarButton>
+
+    <!--After-->
+    <AppBarButton Icon="{ex:SymbolIcon Symbol=Play}"/>
+</CommandBar>
+```
+
+### Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Symbol | Symbol | The `Symbol` representing the icon to display. |
+| FontSize | double | The size of the icon to display. |
+| FontWeight | FontWeight | The thickness of the icon glyph. |
+| FontStyle | FontStyle | The font style for the icon glyph. |
+| IsTextScaleFactorEnabled | bool | Indicates whether automatic text enlargement, to reflect the system text size setting, is enabled. |
+| MirroredWhenRightToLeft | bool | Indicates whether the icon is mirrored when the flow direction is right to left. |
+
+> [!NOTE]
+> The `SymbolIconExtension` actually returns a `FontIcon` value instead of a `SymbolIcon` one. This is done to include the additional properties (eg. `FontSize`, `FontWeight`, etc.) that would otherwise not have been available. When those are not modified, the look of the resulting icon will still be the same as the one that would've resulted from the use of a `SymbolIcon` instance.
+
+## SymbolIconSource
+The [SymbolIconSource markup extension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.symboliconsourceextension) is an alternative for `FontIconSourceExtension` that takes a `Symbol` value instead of a text, and displays the icon with the "Segoe MDL2 Assets". It's equivalent to the `SymbolIconExtension` type, except for the fact that it returns a [`FontIconSource`](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.fonticonsource) instance.
+
+### Syntax
+
+**XAML**
+
+```xml
+<SwipeItems Mode="Reveal">
+    <SwipeItem Text="Play" IconSource="{ex:SymbolIconSource Symbol=Play}"/>
+</SwipeItems>
+```
+
+### Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Symbol | Symbol | The `Symbol` representing the icon to display. |
+| FontSize | double | The size of the icon to display. |
+| FontWeight | FontWeight | The thickness of the icon glyph. |
+| FontStyle | FontStyle | The font style for the icon glyph. |
+| IsTextScaleFactorEnabled | bool | Indicates whether automatic text enlargement, to reflect the system text size setting, is enabled. |
+| MirroredWhenRightToLeft | bool | Indicates whether the icon is mirrored when the flow direction is right to left. |
+
+## Requirements
+
```

Comment:
Would this be a good place to call out the WUX vs MUX compatibility? or up in the intro?

Reply:
Added the note in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/313/commits/316c99e5a331b2725c569eadae439bdeb9c61c29, is this alright?

Reply:
@Sergio0694 looks good. Think we should call out our WinUI 3 preview as then it doesn't matter? We could link to https://aka.ms/wct-winu3. As otherwise I don't think we'll support this scenario until then?

Reply:
That links only refers me to www.microsoft.com, is that aka not active yet?

Reply:
Just a typo (missing i): https://aka.ms/wct-winui3